### PR TITLE
feat(react-components): add a sitewise connected chart story

### PR DESCRIPTION
**Pull request by jmbuss**
## Overview
Adding a story for base chart that uses a real sitewise query.

Adds utils for creating the query with the correct clients.

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

**Review by chejimmy — APPROVED** [review]
Nice! Fine addition to the collection!

File: packages/react-components/.env.example [hunk 1]
```diff
@@ -0,0 +1,9 @@
+AWS_ACCESS_KEY_ID=xxxxxxxxxx
```
Comment (chejimmy) [inline packages/react-components/.env.example:1]: instruction/guide to setup the env?

Comment (mnischay) [reply packages/react-components/.env.example:1]: +1

**Review by mnischay — APPROVED** [review]

**Review by tracy-french — APPROVED** [review]
Great stuff!

File: packages/react-components/.env.example (hunk 1 above)
Comment (jmbuss) [reply packages/react-components/.env.example:1]: yup, these are the variables you need to run that story!

**Pull request merged**